TestVectorsAwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```diff
@@ -67,6 +66,10 @@ dependencies {
     implementation("org.dafny:DafnyRuntime:${dafnyVersion}")
     implementation("software.amazon.smithy.dafny:conversion:0.1")
     implementation("software.amazon.cryptography:aws-cryptographic-material-providers:1.0.1")
```

Comment:
Question: Why not SNAPSHOT for the MPL?

Reply:
Because the goal of the TestVectors is to be published and match the MPL version.
This means that for local development it gets its settings from `settings.gradle.kts`
But it can eventually be run and is explicitly bound to a specific version of the MPL